corehq/warehouse/management/commands/create_batch.py

```diff
@@ -16,11 +20,12 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('dag_slug')
+        parser.add_argument('end_date', nargs='?', default=None)
```

Comment:
should we just make this be required or is there somewhere it's not used?

also we could just pass in type=[utc_timestamp](https://github.com/dimagi/commcare-hq/blob/master/corehq/util/argparse_types.py#L6) and it will automatically handle validation

Reply:
fixed